### ISSUE TITLE
tcp-multiplexer won't start

### ISSUE THREAD

**Issue by Andy200877** [IngmarStein/tcp-multiplexer#4]
i use actual mod-busproxy, but the connection is lost every few hours.
so i want to test your programm as modbus proxy.

but when i start the programm

andy@ModbusProxy:~$ sudo tcp-multiplexer server -p modbus -t 192.168.5.222:501 -l 502
^X^CINFO[2024-08-06T10:05:46+02:00] closing server...
ERRO[2024-08-06T10:05:46+02:00] accept tcp [::]:8000: use of closed network connection
INFO[2024-08-06T10:05:46+02:00] no more connections will be accepted
INFO[2024-08-06T10:05:46+02:00] incoming connections closed
INFO[2024-08-06T10:05:46+02:00] multiplexer server stopped gracefully
INFO[2024-08-06T10:05:46+02:00] server is closed gracefully

i became the error message "use of closed network connection"
when i try with 127.0.0.1:501 the programm starts.
192.168.5.222 is my SolarEdge Solar.

when i ping the ip 192.168.5.222 ist works also.

**Comment by IngmarStein** [comment]
It looks like the multiplexer is actually running until you send it a SIGINT (^C). The error message in this case is harmless (I've removed it in https://github.com/IngmarStein/tcp-multiplexer/commit/048cf66ba13db07dd74723690891c2f9e32b841d) - it essentially just confirms that the connection has been closed during graceful shutdown.

The multiplexer doesn't print anything during normal startup. Did you try if it actually works for you?
